rbac/server/api/roles.py

```diff
@@ -114,24 +112,7 @@
 @ROLES_BP.delete("api/roles/<role_id>/admins")
 @authorized()
 async def delete_role_admin(request, role_id):
-    required_fields = ["id"]
-    utils.validate_fields(required_fields, request.json)
-
-    txn_key = await utils.get_transactor_key(request)
-    proposal_id = str(uuid4())
-    batch_list, _ = role_transaction_creation.propose_remove_role_admins(
-        txn_key=txn_key,
-        batch_key=request.app.config.BATCHER_KEY_PAIR,
-        proposal_id=proposal_id,
-        role_id=role_id,
-        user_id=request.json.get("id"),
-        reason=request.json.get("reason"),
-        metadata=request.json.get("metadata"),
-    )
-    await utils.send(
-        request.app.config.VAL_CONN, batch_list, request.app.config.TIMEOUT
-    )
-    return json({"proposal_id": proposal_id})
+    raise ApiNotImplemented()
```

Comment:
is this going to break ui?

Reply:
Removes the "removal proposal" transactions that
are deprecated and were never completed. Removals
do not require a proposal, are immediate, and will be
re-implemented with new endpoints.